src/credentials-manager/index.ts

```diff
@@ -11,10 +11,7 @@ class CredentialsManager {
   private Auth0Module: Auth0Module;
 
   /**
-   * Construct an instance of CredentialsManager
-   *
-   * @param {String} domain - required - the domain of the credentials to be managed
-   * @param {String} clientId - required - clientId of the credentials to be managed
+   * @ignore
```

Comment:
Ignored so that the constructor does not appear in the published docs (this is always created internally and we don't export the constructor).